wrapanapi/systems/openstack.py

```diff
@@ -758,8 +801,35 @@ def remove_tenant(self, tenant_name):
         tid = self._get_tenant(name=tenant_name)
         self.tenant_api.delete(tid)
 
-    def create_vm(self):
-        raise NotImplementedError('create_vm not implemented.')
+    def create_vm(self, name, image_name=None, image_id=None, assign_floating_ip=False,
```

Comment:
honestly, I'd put this in image.deploy and make this create_vm just alias to image.deploy()

Reply:
The purpose of this function is a bit different from just image deploy, it court circuit the deployment of the image from image name or image id and assign floating ip is needed, to have a ready accessible vm. But personnally prefer the more general connection.create_server that has more options (name or id of different components) usage in this regards that was implemented in earlier commits.

Reply:
actually image.deploy() does the same. I understand your confusion. This is called so due to historical reasons. either way, I don't make you move this code there.

Reply:
I think we can implement for the rest of the system modules by having create_vm implemented to pull the template object and deploy it.  I think the implementation here is acceptable, and if we would prefer the functionality in the deploy method we can move it there when standardizing implementation of create_vm.